kolibri/utils/options.py

```diff
@@ -498,6 +498,14 @@ def url_prefix(value):
                 data that is returned to our telemetry server.
             """,
         },
+        "MINIMUM_DISK_SPACE": {
+            "type": "integer",
+            "default": 250000000,
+            "description": """
+                The minimum free disk space in bytes that Kolibri should try to maintain on the device. This will
+                be used as the floor value to prevent Kolibri completely filling the disk during file import.
+            """,
+        },
```

Comment:
could we define this in MB or GB to be a bit more user-friendly?

Reply:
My main concern would be the frequent confusion between GiB and GB, and the fact that these may be used differently when people are examining their local disks. Using bytes is unambiguous if somewhat verbose.

Reply:
For what it's worth:

* Android [appears to report decimal](https://android.stackexchange.com/a/129682)
* MacOS [reports decimal](https://support.apple.com/en-us/HT201402)
* Ubuntu [shows decimal in the GUI](https://askubuntu.com/a/860890) and `df` can show either

Windows is the only one that seems to [still report base2](https://support.hp.com/lt-en/document/ish_3046042-2787120-16)